### PR TITLE
new(driver,userspace/libsinsp): introduce a new PPME_SYSCALL_BPF_2_ events to avoid using the dynamic argument

### PR DESCRIPTION
**Pull request by FedeDP**
**What type of PR is this?**

> Uncomment one (or more) `/kind <>` lines:

/kind bug

> /kind cleanup

> /kind design

> /kind documentation

> /kind failing-test

/kind feature

**Any specific area of the project related to this PR?**

> Uncomment one (or more) `/area <>` lines:

> /area API-version

> /area build

/area driver-kmod

/area driver-bpf

> /area driver-modern-bpf

> /area libscap-engine-bpf

> /area libscap-engine-gvisor

> /area libscap-engine-kmod

> /area libscap-engine-modern-bpf

> /area libscap-engine-nodriver

> /area libscap-engine-noop

> /area libscap-engine-source-plugin

> /area libscap-engine-udig

> /area libscap

> /area libpman

> /area libsinsp

> /area tests

> /area proposals

**Does this PR require a change in the driver versions?**

> /version driver-API-version-major

> /version driver-API-version-minor

> /version driver-API-version-patch

> /version driver-SCHEMA-version-major

/version driver-SCHEMA-version-minor

> /version driver-SCHEMA-version-patch

I think only the SCHEMA minor version has changed:
* driver sends same data (1 argument for the exit syscall) but with different meaning
* driver sends PPME_SYSCALL_BPF_2_ events, instead of old ones

**What this PR does / why we need it**:

See issues like #454.
Get rid of the "res_or_fd" param for bpf syscall event, instead using just "fd" like we do for, eg: open syscall.

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
new: added a PPME_SYSCALL_BPF_2_ event to get rid of dynamic parameter.
```

### CODE REVIEW

**Comment by jasondellaluce** [comment]
This will need to be ported to the modern eBPF probe too, right? @FedeDP, @Andreagit97

**Comment by FedeDP** [comment]
Yep i mean, as soon as modern bpf supports `bpf` syscall, we should use the newest schema :)

File: userspace/libsinsp/examples/util.cpp [hunk 1]
```diff
@@ -203,7 +203,9 @@ std::string get_event_type(uint16_t type)
         // Process syscalls
         //      
         case PPME_SYSCALL_BPF_E:
-        case PPME_SYSCALL_BPF_X: return "bpf";
+	case PPME_SYSCALL_BPF_2_E:
+        case PPME_SYSCALL_BPF_X:
+	case PPME_SYSCALL_BPF_2_X: return "bpf";
```
Comment (Andreagit97) [inline userspace/libsinsp/examples/util.cpp:208]: they are not aligned but feel free to ignore them if you have no time!

**Review by Andreagit97 — APPROVED** [review]
Thank you very much for this !!!
/approve

File: userspace/libsinsp/examples/util.cpp (hunk 1 above)
Comment (Andreagit97) [reply userspace/libsinsp/examples/util.cpp:208]: To not overload the CI we can avoid to do it here

**Review by leogr — APPROVED** [review]

**Pull request merged**